update_bench_traces_rel_assert.py

```diff
@@ -9,13 +9,20 @@
 from gobexec.model.tools import ExtractTool
 from gobexec.output.renderer import FileRenderer, ConsoleRenderer, MultiRenderer
 
+goblint_assert = GoblintTool(
+    name="goblint_assert",
+    program=str(Path("../analyzer/goblint").absolute()),
+    args=["-v", "--conf", str(Path("../analyzer/conf/traces-rel.json").absolute()), "--enable", "dbg.debug","--set", "trans.activated[+]", "assert"],

```

Comment:
Adding these arguments for assert instrumentation should generate similar asserts:
```console
--set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.relation.privatization mutex-meet-tid-cluster12
```